README.md

```diff
@@ -332,6 +332,7 @@ Table of Contents
   * [numlookupapi.com](https://numlookupapi.com) - Free phone number validation API - 100k free requests / month.
   * [Volca](https://volca.io#api) - Free API providing lists of technologies such as programming languages and database systems. Unlimited free requests.
   * [Query.me](https://query.me) - Collaborative data notebooks that execute in a script-like fashion and allow to fetch and send data via SQL, API, and a growing number custom blocks, like Slack and Email. Free for small Teams.
+  * [Zipcodestack](https://zipcodestack.com) - Free Zip Code API, covering postal codes of all countries. 10000 free requests/month.
```

Comment:
I go the landing page, enter a post code and do search...nothing happens?

Sorry this is not a site thats ready for prime time

Reply:
You are right - things were not quite ready, sorry about that. The search is now working as intended, the API has been running for a while.

Reply:
none of these documentation links work.

Sorry, we're not your QA department, happy to let you know my hourly rate for this kind of work but expecting people to QA your product in a pull request is not appropriate.

We'll close this PR now.

<img width="1792" alt="Screenshot 2022-12-19 at 20 50 41" src="https://user-images.githubusercontent.com/82342/209770803-f21ff369-06dd-461e-8663-242dd3f42171.png">